sys/net/gnrc/network_layer/sixlowpan/frag/sfr/gnrc_sixlowpan_frag_sfr.c

```diff
@@ -483,6 +483,10 @@ void gnrc_sixlowpan_frag_sfr_arq_timeout(gnrc_sixlowpan_frag_fb_t *fbuf)
 
     DEBUG("6lo sfr: ARQ timeout for datagram %u\n", fbuf->tag);
     fbuf->sfr.arq_timeout_event.msg.content.ptr = NULL;
+    if (IS_ACTIVE(CONFIG_GNRC_SIXLOWPAN_SFR_MOCK_ARQ_TIMER)) {
+        /* time goes by ;-) */
+        now -= (fbuf->sfr.arq_timeout * US_PER_MS) + 1;
```

Comment:
What does that mean? 
increment the clock by at least 1 ms - but how does the `arq_timeout` come into play here?

Reply:
This is for the ARQ timer _mock_. Will add a comment.